app/services/reports/adult_incontinence_report_service.rb

```diff
@@ -91,5 +92,55 @@ def donated_supplies
                                     .where(itemizable: organization.donations.for_year(year))
                                     .sum(:quantity)
     end
+
+    def distributed_adult_incontinence_items_from_kits
+      organization_id = @organization.id
+      year = @year
+
+      sql_query = <<-SQL
+        SELECT SUM(line_items.quantity * kit_line_items.quantity)
+        FROM distributions 
+        INNER JOIN line_items ON line_items.itemizable_type = 'Distribution' AND line_items.itemizable_id = distributions.id 
+        INNER JOIN items ON items.id = line_items.item_id 
+        INNER JOIN kits ON kits.id = items.kit_id 
+        INNER JOIN line_items AS kit_line_items ON kits.id = kit_line_items.itemizable_id
+        INNER JOIN items AS kit_items ON kit_items.id = kit_line_items.item_id
+        INNER JOIN base_items ON base_items.partner_key = kit_items.partner_key 
+        WHERE distributions.organization_id = ?
+          AND EXTRACT(year FROM issued_at) = ?
+          AND LOWER(base_items.category) LIKE '%adult%'
+          AND NOT (LOWER(base_items.category) LIKE '%cloth%' OR LOWER(base_items.name) LIKE '%cloth%')
+          AND kit_line_items.itemizable_type = 'Kit';
+      SQL
+
+      sanitized_sql = ActiveRecord::Base.send(:sanitize_sql_array, [sql_query, organization_id, year])
+
+      result = ActiveRecord::Base.connection.execute(sanitized_sql)
+
+      result.first['sum'].to_i
+    end
+
+    def adults_served_per_month
+      (total_people_served_with_loose_supplies_per_month + total_people_served_with_supplies_from_kits_per_month) / 12
+    end
+
+    def total_people_served_with_loose_supplies_per_month
+      organization
+        .distributions
+        .for_year(year)
+        .joins(line_items: :item)
+        .merge(Item.adult_incontinence)
+        .sum('line_items.quantity / COALESCE(items.distribution_quantity, 50)')
+    end # returing 100
+
+    def total_people_served_with_supplies_from_kits_per_month
+      organization
+        .distributions
+        .for_year(year)
+        .joins(line_items: {item: :kit})
+        .merge(Item.adult_incontinence)
+        .where.not(items: {kit_id: nil})
+        .sum('line_items.quantity / COALESCE(items.distribution_quantity, 1)')
+    end
```

Comment:
I *suspect* this is giving you kits that are identified as adult_incontinence, rather than kits that have adult incontinence items in them?